src/components/ListItemCard.tsx

```diff
@@ -1,16 +1,23 @@
+import Image from "next/image";
+import clsx from "clsx";
+import { FC } from "react";
+import { Button } from "@/components/boilerplate/Button";
 
-'use client';
-import {Button} from '@/components/boilerplate/Button';
-import Image from 'next/image'
-import { ItemAddedPreviewModal } from './ItemAddedPreviewModal';
+interface Props {
```

Comment:
Prefer using `interface Props` for components instead of `any` as it gives a clearer scope of the component and does not sacrifice on speed of development